build/packages/rpm/server/SPECS/percona-qan-api2.spec

```diff
@@ -37,18 +37,18 @@ mv %{repo}-%{commit} src/%{provider}
 
 
 %build
-export GOPATH=$(pwd)/
+export GOPATH=$(pwd)/qan-api2
```

Comment:
Seems we shouldn't change this line. And also seems it won't be used in our case at all.

Reply:
Removed.